docs/notebooks/03_merge.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    adata = anndata.concat(mapped_adata, join="outer", fill_value=0)
</pre><p>Consider using <code>index_unique="_"</code> here to ensure that obs names are unique and include the dataset id.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>just forgot to have it in the last version as I was playing around with the <code>concat</code> params. Was not working with the old anndata ...</p>